### PR TITLE
fix(flow): fix editing content elements using action modal

### LINKED ISSUE #75: [BUG] Updating Image Content Image UI issue
**Describe the bug**
image
When updating url in an Image Content's Image input, you can click the Submit button without actually updating the field.

**To Reproduce**

1. Go to flow page
2. Create a node
3. Add an action, Image type, with image and title. 
4. Go back to action, and change the image text. Click Submit, not OK (see the image)

**Expected behavior**
There should be a confirmation pop-up, or the submit button should be disabled (due to no changes being detected), or the change should go through.

**Environment (please complete the following information):**

- OS: macOS
- Browser Chrome
- Browser Version Version 92.0.4515.131 (Official Build) (x86_64)
- Botpress Version 12.25.0
- NodeJS 12.13.0

This causes frustration that happens when you think you updated something and you didn't.

### PR DESCRIPTION
**Pull request by laurentlp**
This PR fixes an issue where when editing content elements (image, video, audio, file) it was possible to save an empty value.

**Before**

https://user-images.githubusercontent.com/9640576/134262576-135a095a-18ca-4b98-8847-807cbf3b9ec4.mp4

**After**

https://user-images.githubusercontent.com/9640576/134262608-e202ba5c-e5dc-4709-9cbc-43e433f12699.mp4

Closes #75 and DEV-1636

### CODE REVIEW

File: packages/studio-ui/src/web/views/FlowBuilder/nodeProps/TransitionSection.jsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default class TransitionSection extends Component {
       <Fragment>
         <div>
           {items.map((item, i) => (
-            <Fragment>
+            <Fragment key={`${i}.${item.node || '-'}`}>
               <ConditionItem className={style.item} condition={item} position={i} key={`${i}.${item.node || '-'}`}>
```
Comment (allardy) [inline packages/studio-ui/src/web/views/FlowBuilder/nodeProps/TransitionSection.jsx:90]: why is the same key used twice ? I thought we only needed it once, also aren't fragments not displayed in the code?

File: packages/studio-ui/src/web/views/FlowBuilder/nodeProps/ActionSection.jsx [hunk 2]
```diff
@@ -118,9 +114,12 @@ export default class ActionSection extends Component {
...
           {items.map((item, i) => (
-            <Popover interactionKind={PopoverInteractionKind.HOVER} position={PopoverPosition.BOTTOM}>
-              <ActionItem className={style.item} text={item} key={`${i}.${item}`}>
-              </ActionItem>
+            <Popover
+              interactionKind={PopoverInteractionKind.HOVER}
+              position={PopoverPosition.BOTTOM}
+              key={`${i}.${item}`}
+            >
+              <ActionItem className={style.item} text={item} key={`${i}.${item}`}></ActionItem>
```
Comment (allardy) [inline packages/studio-ui/src/web/views/FlowBuilder/nodeProps/ActionSection.jsx:122]: key should be unique

File: packages/studio-ui/src/web/views/FlowBuilder/nodeProps/TransitionSection.jsx (hunk 1 above)
Comment (allardy) [inline packages/studio-ui/src/web/views/FlowBuilder/nodeProps/TransitionSection.jsx:90]: same here, unique keys

Comment (laurentlp) [reply packages/studio-ui/src/web/views/FlowBuilder/nodeProps/TransitionSection.jsx:90]: My mistake, I should have removed the one on the ConditionItem component. Indeed Fragments are not displayed, but somehow, they can contain a key and their doc also show an example where the key must be on the fragment when mapping a collection (https://reactjs.org/docs/fragments.html#keyed-fragments)

**Review by EFF — APPROVED** [review]
LGTM

**Pull request merged**